traffic_router/core/src/main/webapp/WEB-INF/applicationContext.xml

```diff
@@ -267,6 +267,7 @@
 
 	<bean id="NameServer" class="org.apache.traffic_control.traffic_router.core.dns.NameServer">
 		<property name="trafficRouterManager" ref="trafficRouterManager" />
+		<property name="negativeCachingTTL" value="900" />
```

Comment:
This is on the right track. Since `applicationContext.xml` is not configurable (not [one of the files](https://traffic-control-cdn.readthedocs.io/en/latest/admin/traffic_router.html#id9) we list for configuration), this should be a property in one of the configuration files, maybe `dns.properties`, then referenced in `applicationContext.xml`.